django-stubs/contrib/auth/base_user.pyi

```diff
@@ -32,6 +32,8 @@ class AbstractBaseUser(models.Model):
     def set_unusable_password(self) -> None: ...
     def has_usable_password(self) -> bool: ...
     def get_session_auth_hash(self) -> str: ...
+    def get_session_auth_fallback_hash(self) -> str: ...
+    def _get_session_auth_hash(self, secret: str | bytes | None = ...) -> str: ...
```

Comment:
We don't include protected methods into stubs.

Reply:
I deleted the part that applies! Thank you